pkg/render/guardian_test.go

```diff
@@ -180,7 +180,7 @@ var _ = Describe("Rendering tests", func() {
 		Expect(component.ResolveImages(nil)).To(BeNil())
 		resources, _ := component.Objects()
 
-		role := rtest.GetResource(resources, "tigera-guardian", "", "rbac.authorization.k8s.io", "v1", "ClusterRole").(*rbacv1.ClusterRole)
+		role := rtest.GetResource(resources, "guardian", "", "rbac.authorization.k8s.io", "v1", "ClusterRole").(*rbacv1.ClusterRole)
```

Comment:
I think global scoped resoures should keep the prefix.

Reply:
Thanks for the spot, updated it to be "calico-" instead (as per the conversations outside of the PR).